<?xml version="1.0" encoding="utf-8"?>
<testsuite failures="0" name="SoftwareManagement.sm_mapper_fail_and_reconnect" skipped="1" tests="1" time="0.01" timestamp="2022-03-21T16:58:43">
	<testcase classname="MapperC8yReceiveLastMessageOnRestart" name="SoftwareManagement.sm_mapper_fail_and_reconnect">
		<skipped message="SKIPPED: instable https://github.com/thin-edge/thin-edge.io/issues/752"/>
		<system-out>16:58:43 DEBUG deletedir failed to update permissions and delete /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_mapper_fail_and_reconnect/Output/linux on this attempt: [Errno 2] No such file or directory: '/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_mapper_fail_and_reconnect/Output/linux'
16:58:43 INFO  ==============================================================
16:58:43 INFO  Id:    SoftwareManagement.sm_mapper_fail_and_reconnect
16:58:43 INFO  Dir:   software_management_end_to_end/
16:58:43 INFO  Title: Validate the tedge-mapper-sm-c8y for receiving the last message on restart
16:58:43 DEBUG Execution order hint: 0.0
16:58:43 INFO  ==============================================================
16:58:43 INFO  instable https://github.com/thin-edge/thin-edge.io/issues/752 ... skipped
16:58:43 DEBUG --- test cleanup
16:58:43 DEBUG ProcessUser cleanup function done.
16:58:43 INFO  
16:58:43 INFO  Test duration: 0.01 secs
16:58:43 INFO  Test final outcome:  SKIPPED
16:58:43 INFO  Test outcome reason: instable https://github.com/thin-edge/thin-edge.io/issues/752
16:58:43 INFO  
</system-out>
	</testcase>
</testsuite>
